tests/plugins/pastefromword/generated/onenote.js

```diff
@@ -0,0 +1,38 @@
+/* bender-tags: clipboard,pastefromword */
+/* jshint ignore:start */
+/* bender-ckeditor-plugins: pastefromword,ajax,basicstyles,toolbar */
+/* jshint ignore:end */
+/* bender-include: _lib/q.js,_helpers/promisePasteEvent.js,_helpers/assertWordFilter.js,_helpers/createTestCase.js */
+/* bender-include: _helpers/createTestSuite.js */
+/* global createTestSuite */
+
+( function() {
+	'use strict';
+
+	var config = {
+		language: 'en',
+		colorButton_normalizeBackground: false,
+		allowedContent: 'p ul li'
```

Comment:
Rather than define explicitly allowedContent, just add `list` plugin, and that will be fine.